starboard/nplb/nplb_evergreen_compat_tests/fonts_test.cc

```diff
@@ -45,6 +52,9 @@ TEST(FontsTest, VerifySystemFontsDirectory) {
 }
 
 TEST(FontsTest, VerifySystemFontsConfigDirectory) {
+#if !SB_IS(EVERGREEN)
+  GTEST_SKIP() << "Only full Evergreen platforms support this test.";
+#endif
   std::vector<char> system_fonts_conf_dir(kSbFileMaxPath);
   ASSERT_TRUE(SbSystemGetPath(kSbSystemPathFontConfigurationDirectory,
```

Comment:
@y4vor the three failing tests verify that a given directory exists in the system (CrashpadConfigTest.VerifyUploadCert also verifies that a named file exist in the folder). We could make those magically appear in bots, but devs running these locally would most likely fail. Hence my approach was to skip those tests in e.g. desktop Linux, raspberry Pi etc (but not on RDK). Any ideas how to do that via either buildflags or runtime feature detection?